src/modules/reporting/components/reporting-dispute-form/reporting-dispute-form.jsx

```diff
@@ -258,7 +265,7 @@ export default class ReportingDisputeForm extends Component {
       }
     }
 
-    ReportingDisputeForm.checkStake(stake, updatedValidations)
+    ReportingDisputeForm.checkStake(stake, updatedValidations, this.calculateMaxRep())
```

Comment:
Is it possible for this.state.selectedOutcome to exist here? If so, should it be passed in?